backend/server.py

```
@@ -2862,18 +2897,19 @@ async def _run_agent_loop(
     try:
-        await get_db().activity_log.insert_one(
-            {
-                "category": category,
-                "message": message,
-                "user_id": user_id,
-                "meta": meta or {},
-                "created_at": datetime.now(timezone.utc).isoformat(),
-            }
-        )
+        await get_db().activity_log.insert_one(dict(entry))
     except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)